samples/Dantooine/Dantooine.Client/README.md

```diff
@@ -0,0 +1,58 @@
+# Built using Blazor.BFF.OpenIDConnect.Template
```

Comment:
I suspect this won't age well as it's the type of things we tend to forget after a few releases 😄 
Please remove it and consider adding a code comment with a link to your repository in Program.cs if you think it's interesting to mention the fact a template exists.

Reply:
removed file